WordPress/src/main/java/org/wordpress/android/AppInitializer.kt

```diff
@@ -890,6 +890,7 @@ class AppInitializer @Inject constructor(
         private const val DEFAULT_TIMEOUT = 2 * 60 // 2 minutes
 
         @SuppressLint("StaticFieldLeak") var context: Context? = null
+            private set;
```

Comment:
Minor (🔍): This is a Kotlin file and as such consider omitting the trailing `;` at the end of the statement.

Reply:
Fixed in bae3f3ee179ddd234a3b7bbcf4c18eb9908a9f28